config/helpers.exs

```diff
@@ -27,16 +27,20 @@ defmodule Cog.Config.Helpers do
   end
 
   # Returns a list containing the enabled chat provider and all other providers.
-  def provider_list,
-    do: [Enum.find(@chat_providers, &enabled_provider?/1)] ++ @other_providers
+  # Or nil if no provider is specified.
+  def provider_list do
+    case Enum.find(@chat_providers, &enabled_provider?/1) do
+      nil -> nil
```

Comment:
I set `nil` because that's what it would be if it wasn't set. And that's what the `Cog.Chat.Adapter` was expecting. We could change it to `[]` if that would be clearer, but it would also mean a change in the adapter.